cmake/Modules/FindSphinx.cmake

```diff
@@ -354,12 +354,18 @@ function(sphinx_add_docs _target)
 
   set(_cachedir "${CMAKE_CURRENT_BINARY_DIR}/${_target}.cache")
   file(MAKE_DIRECTORY "${_cachedir}")
-  file(MAKE_DIRECTORY "${_cachedir}/_static")
+  if(EXISTS "${_sourcedir}/_static")
+    file(COPY "${_sourcedir}/_static" DESTINATION "${_cachedir}")
+  endif()
```

Comment:
Should we add an `else()` that does `MAKE_DIRECTORY`?

Reply:
Sure. Pushed.